JobConfig/primary/prolog.fcl

```diff
@@ -22,6 +22,7 @@ Primary: {
     @table::TrackerMC.StepProducers
     @table::CaloMC.StepProducers
     CrvSteps : @local::CrvSteps
+    compressPV: @local::Cosmic.compressPV
```

Comment:
I see it's something you added.  primary/prolog should not instantiate a module from Cosmic, that's a reverse-dependency.  Please move this to cosmic/prolog.fcl